### PR TITLE
Fix: Firefox OS area link

### LINKED ISSUE #5024: Issue with "Firefox OS": Dead Firefox OS area link
MDN URL: https://developer.mozilla.org/en-US/docs/Glossary/Firefox_OS

#### What information was incorrect, unhelpful, or incomplete?

The Firefox OS area is dead.

#### Specific section or headline?

Technical reference

#### What did you expect to see?

A link into the archive?

#### Did you test this? If so, how?

I clicked the link.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/glossary/firefox_os`
* MDN URL: https://developer.mozilla.org/en-US/docs/Glossary/Firefox_OS
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/glossary/firefox_os/index.html
* Last commit: https://github.com/mdn/content/commit/cbe151a06d6e5b4d1fbb46081bd16e69ef4c1630
* Document last modified: 2020-09-15T15:48:51.000Z

</details>

### PR DESCRIPTION
**Pull request by Blakelist7**
Signed-off-by: Kanwal Singh <kanwal.88singh@gmail.com>

> What was wrong/why is this fix needed? (quick summary only)
Dead Link
> MDN URL of the main page changed

> Issue number (if there is an associated issue)
fixes: #5024 
> Anything else that could help us review it

### CODE REVIEW

**Comment by wbamberg** [comment]
We shouldn't link to pages in the archive, and this entry is written as if Firefox OS is still a thing. I think this entry should be rewritten to say something like:

> Firefox OS is a discontinued mobile operating system developed by Mozilla. See also [Firefox OS](https://en.wikipedia.org/wiki/Firefox_OS) Wikipedia page for more details.

...and we should do something similar for the Gaia and Gonk pages too.

**Comment by Blakelist7** [comment]
Yeah, this sounds good.

**Comment by Blakelist7** [comment]
BTW could you please tell why is the Archive link  (`/en-US/docs/Archive/B2G_OS`) is throwing "no page found" error, as I checked there were some more pages linked to Archive Repo and they were written in the same manner but where working fine.

**Comment by wbamberg** [comment]
> BTW could you please explain why is Archive link (`/en-US/docs/Archive/B2G_OS`) throwing "no page found" error, as I checked there were some more pages linked to Archive Repo and they were written in the same manner but where working fine.

Sorry, I have no idea about that. But I think linking to Archive pages from anywhere is always suspect. The Archive, by definition, is the place we put stuff that's not useful or relevant any more, so we should not be linking there.

**Comment by Ryuno-Ki** [comment]
@wbamberg I answered the question regarding broken link in the related issue.
I leave the decision about the fix in this PR to you.

File: files/en-us/glossary/firefox_os/index.html [hunk 1]
```diff
@@ -9,9 +9,9 @@
   - Infrastructure
   - Intro
 ---
-<p id="Summary">Firefox OS is Mozilla's mobile operating system, based on Linux and {{glossary("Mozilla Firefox","Firefox's")}} powerful {{glossary("Gecko")}} rendering engine. Firefox OS consists mainly of {{glossary("Gaia")}}, {{glossary("Gecko")}}, and {{glossary("Gonk")}}.</p>
+<p id="Summary">Firefox OS is a discontinued open-source operating system(for smartphones, tablets) developed by Mozilla. Refer {{interwiki("wikipedia", "Firefox OS")}} for more details.</p>
 
-<h2 id="Learn_more">Learn more</h2>
+<!-- <h2 id="Learn_more">Learn more</h2>
```
Comment (Ryuno-Ki) [inline files/en-us/glossary/firefox_os/index.html]: Since we have it in git, you can safely delete this instead of commenting.

Comment (Blakelist7) [reply files/en-us/glossary/firefox_os/index.html]: I have deleted them in Update index.html please check

File: files/en-us/glossary/firefox_os/index.html [hunk 2]
```diff
@@ -9,18 +9,4 @@
...
-<ul>
- <li>{{interwiki("wikipedia", "Firefox OS")}} on Wikipedia</li>
-</ul>
-
-<h3 id="Technical_reference">Technical reference</h3>
-
-<ul>
- <li>Find out much more about Firefox OS in the MDN <a href="/en-US/Firefox_OS">Firefox OS Zone</a>!</li>
-</ul>
+<p id="Summary">Firefox OS is a discontinued open-source operating system(for smartphones, tablets) developed by Mozilla. Refer {{interwiki("wikipedia", "Firefox OS")}} for more details.</p>
```
Comment (wbamberg) [inline files/en-us/glossary/firefox_os/index.html:12]:
```suggestion
<p id="Summary">Firefox OS is a discontinued open source mobile operating system developed by Mozilla. See {{interwiki("wikipedia", "Firefox OS")}} for more details.</p>
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks @Blakelist7 . Just some copy edits. If you feel like doing the same for https://developer.mozilla.org/en-US/docs/Glossary/Gaia and https://developer.mozilla.org/en-US/docs/Glossary/Gonk that would be most helpful too!

**Comment by Blakelist7** [comment]
> Thanks @Blakelist7 . Just some copy edits. If you feel like doing the same for https://developer.mozilla.org/en-US/docs/Glossary/Gaia and https://developer.mozilla.org/en-US/docs/Glossary/Gonk that would be most helpful too!

On it.

**Review by wbamberg — APPROVED** [review]
Thanks @Blakelist7 !

**Pull request merged**